app/src/main/java/ch/epfl/sdp/blindly/helpers/Message.kt

```diff
@@ -10,17 +15,16 @@ class Message {
     // timestamp is to achieve correct ordering of the messages and ensure fetching is in
     // same ordering the next time users open the chat
     var timestamp: Long? = null
-    var messageText: String? = null
+    var messageText: T? = null
```

Comment:
Shouldn't the type to be sent inside the message always be a String?

Reply:
This caught my eyes as well, I think @koko-ng decided to use same object to store locations as well. When I checked the Realtime Database I saw that locations were sent in a messageText and then it had latitude and longitude as child attributes. This might be better in terms of not replicating code and easier to test however it creates a little bit of confusion. Make we can adjust the naming a bit ?

Reply:
No, the LiveLocation sends BlindlyLatLng